README.md

```diff
@@ -47,12 +47,42 @@ You will also need to add `virgil-crypto` and `virgil-sdk` scripts.
 Now you can use global variables `Keyknox`, `Virgil` and `VirgilCrypto` as namespace objects, containing all of `@virgilsecurity/keyknox`, `virgil-sdk` and `virgil-crypto` exports as properties.
 
 ## Usage Example
-You can find an example of simple client-server application [here](example).
+To begin using Virgil Keyknox SDK you'll need to initialize `SyncKeyStorage` class. This class is responsible for synchronization between Keychain and Keyknox Cloud. In order to initialize `SyncKeyStorage` class you'll need the following values:
```

Comment:
`...between Keychain and Keyknox Cloud` - this should be `between device storage - IndexedDB in a browser or file system in node.js - and Keyknox Cloud`. Keychain is specific to iOS